public/doc/openapi-3-v1.2.json

```diff
@@ -1519,6 +1559,28 @@
                     }
                 }
             },
+            "Reposition": {
+                "description": "The desired increment relative to its current position, or placement to top or bottom of the list.",
+                "type": "object",
+                "properties": {
+                    "increment": {
+                        "type": "integer",
+                        "title": "Increment",
+                        "description": "Move the record up (negative) or down (positive) in the list. Do not set it if placement is used",
+                        "example": -2
+                    },
+                    "placement": {
+                        "type": "string",
+                        "title": "Placement",
+                        "description": "Place the record to the top or bottom of the list. Do not set it if increment is used",
+                        "enum": [
+                            "bottom",
+                            "top"
+                        ],
+                        "example": "top"
```

Comment:
@bzwei Should this also have nullable true.
Also we should be testing that the nullable goes thru the openapi validation. If we do a POST with this data the openapi-validation is going to deal with it first and if the enum is nullable it should bypass it.

I think there has to be a spec at the controller/request level that validates that flow, if the value is "middle" the openapi validator would raise a "Bad Argument" but if it is null it should allow it thru.